tests/auth_tests/test_forms.py

```diff
@@ -439,6 +439,13 @@ class CustomAuthenticationForm(AuthenticationForm):
         username_field = User._meta.get_field(User.USERNAME_FIELD)
         self.assertEqual(form.fields['username'].label, capfirst(username_field.verbose_name))
 
+    def test_username_field_autocapitalize_off(self):
+        data = {
+            'username': 'testclient'
+        }
+        form = AuthenticationForm(None, data)
+        self.assertEqual(form.fields['username'].to_python('testclient'), 'testclient')
```

Comment:
This test passes without changes in the `UsernameField`.

Reply:
Since the behaviour of the `autocapitalize` attribute has nothing to do with django itself, you'll probably have to write some sort of a test that the field/widget renders that attribute.

Reply:
You should just check for the presence of the attribute. See tests in #11070.